src/material/slider/slider-input.ts

```diff
@@ -682,7 +710,10 @@ export class MatSliderRangeThumb extends MatSliderThumb implements _MatSliderRan
       this._slider.min < this._slider.max
         ? (this.max - this.min) / (this._slider.max - this._slider.min)
         : 1;
-    const width = maxWidth * percentage + minWidth;
+    let width = maxWidth * percentage + minWidth;
+    if (this._isSafariIOS) {
+      width -= 2;
```

Comment:
Do we have any idea why this might be? padding, border, margin, outline, box-sizing? It _feels_ like probably just the result of some default CSS styling that we could maybe change instead

Reply:
Looking into this today, I was able to remove this without any issues. Not sure why this was breaking yesterday